features-json/css-touch-action.json

```diff
@@ -47,7 +47,7 @@
       "8":"n",
       "9":"n",
       "10":"y x #2",
-      "11":"y"
+      "11":"n"
```

Comment:
IE10 was the first browser to support pointer events and touch-action, what gives you the impression it disappeared in IE11?